### PR TITLE
Article 119: Supervised Learning

### PR DESCRIPTION
**Pull request by Garcko**
#### Length of text ####
:page_with_curl: short 

####  Difficulty to read ####
 :mouse:  light 

####  Difficulty to understand #### 
 :school_satchel:  easy 

####  Type of text #### 

 :bowtie:  informative 

####  Type of header #### 
 

 :boom:  catchy

### CODE REVIEW

File: articles/article-119-supervised-learning.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Figure
+* A simple classification graph, some points and then the separating line (maybe to difficult for understanding)
+* A teacher computer, who teachs some smaller computer with a blackboard
+
+
+
+<!-- BEGIN OPTIONAL -->
+## Remark
+* Remove the overfitting part.
```
Comment (s3lph) [inline articles/article-119-supervised-learning.md]: Please don't put review notes into the optional text section. Instead, add them as a comment to the pull request.

**Comment by Fjolyo** [comment]
# Plenum Suggestions

- Mention problem instead of example

- Title is not explained enough / confusing

- Maybe mention supervised/unsupervised

- Don't mention "overfitting"

**Comment by MoiraZuber** [comment]
Added an image suggestion.

**Comment by Garcko** [comment]
changed the title, removed remark and figure text,  removed the overfitting part,  I am not sure what to change for "mention problem instead of example"

**Comment by Fjolyo** [comment]
We think the content of the article is good, but the sentences and the grammar needs some adjustments.
Maybe the grammar team can help here.

**Pull request closed without merging**